tools/accuracy_checker/accuracy_checker/annotation_converters/super_resolution_converter.py

```diff
@@ -71,10 +84,31 @@ def parameters(cls):
 
     def configure(self):
         self.data_dir = self.get_value_from_config('data_dir')
+        self.lr_dir = self.get_value_from_config('lr_dir') if self.get_value_from_config('lr_dir') else self.data_dir
+        if not self.lr_dir:
+            raise ConfigError('One of the parameters: data_dir or lr_dir should be provided for conversion')
+        self.hr_dir = self.get_value_from_config('hr_dir') if self.get_value_from_config('hr_dir') else self.data_dir
+        self.upsampled_dir = self.get_value_from_config('upsampled_dir')
+        if not self.upsampled_dir:
```

Comment:
this dir used only if we have two_streams in input, so no need to fill it in other cases